lib/puppet/pops/lookup/lookup_adapter.rb

```diff
@@ -432,8 +432,8 @@ def initialize_env_provider(lookup_invocation)
       ep
     else
       unless Puppet[:strict] == :off
-        msg = "Defining environment_data_provider='#{provider_name}' in environment.conf is deprecated"
-        msg += ". A '#{HieraConfig::CONFIG_FILE_NAME}' file should be used instead" if ep.nil?
+        msg = _("Defining environment_data_provider='%{provider_name}' in environment.conf is deprecated") % { provider_name: provider_name }
```

Comment:
@Magisus environment_data_provider is a keyword and environment.conf refers to a filename, and should be left out of translation correct?